graphql/schema.graphql

```diff
@@ -160,11 +160,15 @@ type Mutation {
 
   """ Profile """
   updateProfile(fields: ProfileInput, customFields: [ProfileCustomFieldInput!]): Profile
-  uploadProfilePicture(file: Upload!, field: String!): ProfileCustomField
   updateAddress(id: String, fields: AddressInput): Profile
   createAddress(fields: AddressInput): Profile
   deleteAddress(id: String): Boolean
 
+  """ Uploads the Profile Picture by ereasing the old ones """
```

Comment:
Uploads the profile picture...
**erasing**

Reply:
Thanks !